### PR TITLE
Create the onboarding screen

### PR DESCRIPTION
**Pull request by yongmin86k**
# Create the onboarding screen

## Used dependencies
- react-final-form
- react-native-check-box

## Change logs
- Update global styles
    - Add font size, weight
    - Add styles for button & button text
- Create Button component
- Create Input Field component
- Create Input Checkbox component

image

### CODE REVIEW

File: client/js/screens/SignIn/SignIn.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                          email: 'Please enter the email',
+                        });
+                      }
+                      if (!values.password) {
+                        setErrors({
+                          ...isErrors,
+                          password: 'Please enter the password',
+                        });
+                      }
+                      // console.log(values, isErrors);
```
Comment (siddharthparmar7) [inline client/js/screens/SignIn/SignIn.js]: Please remove all the console logs from your file.

File: client/js/screens/SignIn/SignIn.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+      <ImageBackground
+        style={styles.signInBgImg}
+        source={require('../../assets/images/imgSignIn01.jpg')}
+      />
+      <ScrollView>
+        <SafeAreaView style={THEME.padding.accountScreen}>
+          <TitleText style={styles.title}>Sign In</TitleText>
+          <Form
+            onSubmit={values => {
+              console.log('submit: ', values);
```
Comment (siddharthparmar7) [inline client/js/screens/SignIn/SignIn.js]: Please implement the auth logic and remove the console log.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please implement the auth logic with react-navigation auth flow.

File: client/js/screens/SignIn/SignIn.js (hunk 2 above)
Comment (yongmin86k) [reply client/js/screens/SignIn/SignIn.js]: I'll create another issue for the auth logic. 
This was only for creating elements of the Sign-in screen.

File: client/js/screens/SignIn/SignIn.js (hunk 1 above)
Comment (yongmin86k) [reply client/js/screens/SignIn/SignIn.js]: Got it

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**